Linphone/App.xaml.cs

```diff
@@ -227,7 +239,24 @@ protected override void OnActivated(IActivatedEventArgs args)
                 var arguments = toastArgs.Argument;
                 Initialize(args, arguments);
             }
+        }
 
+        private async void App_CloseRequested(object sender, SystemNavigationCloseRequestedPreviewEventArgs e)
+        {
+            // Todo: Unregister before shutting down.

```

Comment:
Why does not use deferral as you can see in  https://stackoverflow.com/a/62558006/1539100

Reply:
resolved.